### PR TITLE
 Implement cartographer_grpc_node.

### PR DESCRIPTION
**Pull request by cschuet**

### CODE REVIEW

File: cartographer_ros/cartographer_ros/cartographer_grpc/node_grpc_main.cc [hunk 1]
```diff
@@ -26,7 +28,9 @@ DEFINE_string(configuration_directory, "",
 DEFINE_string(configuration_basename, "",
               "Basename, i.e. not containing any directory prefix, of the "
               "configuration file.");
-DEFINE_string(map_filename, "", "If non-empty, filename of a map to load.");
+DEFINE_string(server_address, "localhost:50051",
```
Comment (gaschler) [inline cartographer_ros/cartographer_ros/cartographer_grpc/node_grpc_main.cc]: Is there a way to get this default value from the original? Defining this string here is likely to get out of sync.

Comment (cschuet) [reply cartographer_ros/cartographer_ros/cartographer_grpc/node_grpc_main.cc]: There no easy was as discussed: on the server side the default is set in the lua file the server options are parsed from. cartographer_node_grpc however is not configured using lua files which is why there is no other place in the C++ code where this constant appears.

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

**Pull request merged**